### ISSUE TITLE
[Visual requirement – Make Code – Import File] Luminosity ratio is less than 3:1 for focus indicator over the 'Choose File' button under 'Import File' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3569]
**User Experience:**
Visual impaired user will find difficulty to see the focus indicator over the control or element and border line for edit field if that focus indicator and border line have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to import file option and select it. Navigate through the all the controls of 'Import File' dialog.
6. Verify contrast ratio of focus indicator over the 'Choose File' button.

**Actual Result:**
Luminosity ratio is less than 3:1 for focus indicator over the 'Choose File' button under 'Import File' pop-up window and due to this, visual impaired user is unable to see the focus indicator.

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for focus indicator over the 'Choose File' button under 'Import File' pop-up window, so that visual impaired user could easily see the border of focus.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.11 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.11, #Win10-Edge(Chromium)

![WCAG1 4 11_Import File_CCA](https://user-images.githubusercontent.com/72871089/97965584-9e42f480-1de0-11eb-99d0-47e3e51f0678.jpg)
